package/piksi_system_daemon/src/main.c

```diff
@@ -141,60 +116,26 @@ static int uart_configure(const uart_t *uart)
       (cfgetospeed(&tio) != baudrate_val_table[uart->baudrate]) ||
       ((tio.c_cflag & CRTSCTS) ? (uart->flow_control != FLOW_CONTROL_RTS_CTS) :
                                  (uart->flow_control != FLOW_CONTROL_NONE))) {
-    printf("error configuring tty\n");
+    piksi_log(LOG_ERR, "error configuring tty");
     return -1;
   }
 
   return 0;
 }
 
-static bool baudrate_notify(struct setting *s, const char *val)
+static int baudrate_notify(void *context)
```

Comment:
nit: `baudrate_notify` and `flow_control_notify` have the same signature and seem to do exactly the same thing, is it worth consolidating to one function maybe `uart_notify`?